parameter/DefaultElementLibrary.h

```diff
@@ -46,7 +47,13 @@ class CDefaultElementLibrary: public CElementLibrary
 {
 public:
 
-    explicit CDefaultElementLibrary(bool bEnableDefaultMechanism = true);
+    /**
+     * Class constructor
+     *
+     * @param[in] logger the logger provided by the client
+     * @param[in] bEnableDefaultMechanism boolean allowing to activate VirtualSubsystem use
+     */
+    explicit CDefaultElementLibrary(core::log::ILogger& logger, bool bEnableDefaultMechanism = true);
```

Comment:
Although it does no harm keeping the `explicit` keyword, I'm wondering (for my own curiosity): is it still needed ? This constructor is no longer a single-argument ctor and can therefore no longer be used for implicit conversion (even though it has an optional argument ?), right ?

Anyway, the 'explicit' keyword should probably be kept here.